src/providers/uploadsManager/useStartFileUpload.tsx

```diff
@@ -44,39 +42,16 @@ export const useStartFileUpload = () => {
   const displayUploadingNotification = useRef(
     debounce(() => {
       displaySnackbar({
-        title:
-          pendingUploadingNotificationsCounts.current > 1
-            ? `${pendingUploadingNotificationsCounts.current} assets being uploaded`
-            : 'Asset being uploaded',
-        iconType: 'info',
+        title: `(${pendingUploadingNotificationsCounts.current}) ongoing upload`,
```

Comment:
I'm not sure this title will work great. This suggest that this is total number of ongoing uploads when in reality it will be only number of items just added. Maybe let's use something like `x file(s) added to uploads`. Wdyt?

Reply:
Good point I didn't notice this when I looked at the code. 100% agree